client/packages/common/src/utils/environment/EnvUtils.ts

```diff
@@ -71,10 +73,14 @@ const getPlatform = () => {
   }
 };
 
+const platform = getPlatform();
+
 export const EnvUtils = {
   // Using isProduction rather than isDevelopment, as we also use a testing
   // environment while running jest, so easier to check !isProduction, generally.
   isProduction: (): boolean => process.env['NODE_ENV'] === 'production',
   mapRoute,
-  platform: getPlatform(),
+  platform,
```

Comment:
I didn't like having the consumer determine this by checking the platform when using. have moved the html/pdf decision into here so that its in one place